src/hotspot/os_cpu/linux_riscv/riscv_hwprobe.cpp

```diff
@@ -175,6 +175,9 @@ void RiscvHwprobe::add_features_from_query_result() {
   if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZBS)) {
     VM_Version::ext_Zbs.enable_feature();
   }
+  if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZVBB)) {
+    VM_Version::ext_Zvbb.enable_feature();
+  }
```

Comment:
I don't think it's appropriate to auto-enable an experimental extension.

Reply:
Thanks for catching, fixed.